src/shape/bounding-box.js

```diff
@@ -75,29 +74,28 @@ Object.assign(BoundingBox.prototype, {
         th.x = (tmaxx - tminx) * 0.5;
         th.y = (tmaxy - tminy) * 0.5;
         th.z = (tmaxz - tminz) * 0.5;
-    },
+    }
 
     /**
      * @function
      * @name pc.BoundingBox#copy
      * @description Copies the contents of a source AABB.
      * @param {pc.BoundingBox} src - The AABB to copy from.
      */
-    copy: function (src) {
+    copy(src) {
         this.center.copy(src.center);
         this.halfExtents.copy(src.halfExtents);
-        this.type = src.type;
```

Comment:
should be ok to remove, I don't see references to it .. but curious what it was?

Reply:
So this used to be a string type property on each shape instance that specified what it was. But it was removed years ago. This bug was actually pointed out to me by the Typescript compiler but since that PR is unlikely to be merged, I'm adding the fix here.